docs/tutorial/fast_data/fast_data_tutorial.mdx

```diff
@@ -24,143 +31,167 @@ Before starting this tutorial, you need to be familiar with the concepts of [Fas
-![Create System of Records](img/05b-page-overview-projection-kafka.png) 
 
 #### Fields
-Now we need to define the fields that we want to store in the Projection.
-![Create Projection Fields](img/06-create-projection-field.png)
 
-Inside this tutorial we will use the following fields:
+Now we need to define the fields that we want to store in the Projection. You can do that by clicking the "Add field" button located at the top-right of the "Fields card".
+
+![Create Projection Fields](img/06-create-projection-field.png)
 
 :::info
-It's mandatory to define at least one field and at last one primary key.
+It is mandatory to define at least one custom field. Also, you must define at least one field as _Primary Key_ (select the related checkbox).
 :::
 
 :::info
-You can leave _defaultIdentity_ as value for Cast function when entering the fields for this tutorial.
+You can leave _defaultIdentity_ as a value for _Cast Function_ when entering the fields for this tutorial. 
+In this way, every value inside the payload received from the Source Systems will be saved in the database without further customizations.
 :::
 
+For the scope of this tutorial, we ask you to add the following fields:
+
 | Field           | Type   | Description                                                       |
 |-----------------|--------|-------------------------------------------------------------------|
-| **registry_id** | String | The registry ID. This field is required and it's the Primary Key. |
+| **registry_id** | String | The registry ID. Check this field as "Required" and "Primary Key".|
 | **name**        | String | The name of the customer to whom the registry refers.             |
 | **email**       | String | The email of the customer to whom the registry refers.            |
 | **phone**       | String | The phone of the customer to whom the registry refers.            |
 | **address**     | String | The address of the customer to whom the registry refers.          |
 | **city**        | String | The city of the customer to whom the registry refers.             |
 | **state**       | String | The State of the customer to whom the registry refers.            |
 
+#### Ingestion Topics
+
+At the bottom of the same page, we can also see the Kafka topics for each environment. There are two types of topics (_Ingestion topics_ and _PR Update topics_) 
+but within the scope of this tutorial, only the ingestion topics are considered. The Kafka Ingestion topics will be used by the Real Time Updater 
+to receive the data from the Source Systems so that the Projections will be updated, but we will discuss this later.
+![Create System of Records](img/05b-page-overview-projection-kafka.png) 
```

Comment:
This might be an hard take, but I prefer to not change the topic names (for now) because I'm assuming that this tutorial might be in a dedicated demo project where the topics are already configured.

Anyway, rn the topics are automatically generated as: company, environment, system, projection name, suffix.
I'd prefer to keep it that way.